io/CMakeLists.txt

```diff
@@ -5,13 +5,21 @@ else()
   add_definitions(-DNO_INLINE_POST)
 endif()
 
+if(PARALLEL_NETCDF)
```

Comment:
file names must be module_write_* not model_write_*

Reply:
FIxed, thanks